RichTextVC-iOS.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = "RichTextVC-iOS"
-  s.version          = "0.1.1"
+  s.version          = "0.1.5"
```

Comment:
Typically, the convention for versioning is this: `major.minor.patch`. So if you're releasing this, I'd start it at `1.0.0`, and work your way up from there. On your next update, go to `1.1`, if its a bug fix (or patch) release, then `1.1.1`.